process/block/metablock.go

```diff
@@ -1185,6 +1185,12 @@ func (mp *metaProcessor) updateState(lastMetaBlock data.HeaderHandler) {
 		return
 	}
 
+	if lastMetaBlock.IsStartOfEpochBlock() {
+		log.Debug("trie snapshot", "rootHash", lastMetaBlock.GetRootHash())
+		mp.accountsDB[state.UserAccountsState].SnapshotState(lastMetaBlock.GetRootHash())
+		mp.accountsDB[state.PeerAccountsState].SnapshotState(lastMetaBlock.GetRootHash())
```

Comment:
lastMetaBlock.GetValidatorStatsRootHash() instead of lastMetaBlock.GetRootHash()

Reply:
thanks.